libs/c2/manifest.yaml

```diff
@@ -0,0 +1,23 @@
+
+info:
+#    name: "C library"
+    language: c
+    type: library
+    kinds:
+        - static
+        - dynamic
```

Comment:
language can be c2, kinds only static. In the future i want to add a 'source ' kind as well for libraries.

Reply:
Good points, I shall remove the `dynamic`.